src/styles/calendar.scss

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1px;
+  padding: 10px;
+  width: calc(7 * ($calendar-day-size + 1px));
+
+  &__day {
+    box-sizing: border-box;
+    width: $calendar-day-size;
+    height: $calendar-day-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translate(0, -20px);
+      transition: 1s ease-in-out 0.5s;
```

Comment:
move transition property to &__day selector. it will make smooth changing styles in two ways.